pay-api/tests/postman/pay-api.postman_collection.json

```diff
@@ -80,89 +76,34 @@
 			"response": []
 		},
 		{
-			"name": "Create a Transaction Record",
```

Comment:
It lost the changes for transactions endpoint on merge. Please re-merge.

Reply:
updated.